lib/index.js

```diff
@@ -5,16 +5,16 @@ const D3Node = require('d3-node');
 
 const styles = `
 .current {
-  fill: #2aa748;
+  fill: #5FA04E;
 }
 .active {
-  fill: #47b4ff;
+  fill: #229AD6;
 }
 .maintenance {
-  fill: #89a19d;
+  fill: #B1BCC2;
 }
 .unstable {
-  fill: #ffb800;
+  fill: #E99C40;
```

Comment:
These changes switch to capital letters for the color codes but leave them lower case everywhere else in the file. Let's just use one or the other rather than mixing.